python/foxglove-sdk/src/lib.rs

```diff
@@ -135,6 +135,14 @@ impl BaseChannel {
         Ok(())
     }
 
+    fn topic(&self) -> &str {
+        self.0.topic()
+    }
+
+    fn schema_name(&self) -> Option<&str> {
+        self.0.schema().map(|s| s.name.as_str())
```

Comment:
Btw you can also use the ? Operator on options if the function returns an option.

No need to change this, just pointing it out in case you didn't know.